.vale.ini

```diff
@@ -16,4 +16,4 @@ Datadog.latin = NO
 Datadog.gender = NO
 Datadog.emdash = NO
 Datadog.dashes = NO
-Datadog.americanspelling = NO
+Datadog.americanspelling = NO
```

Comment:
I don't think this is a real change

Reply:
Just removed a newline at the end of file, all good 👍